src/build.props

```diff
@@ -11,7 +11,7 @@
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
     
     <!-- VersionPrefix denotes the current Semantic Version for the SDK. Must be updated before every nuget drop. -->
-    <VersionPrefix>2.1.3</VersionPrefix>
```

Comment:
>2.1.3 [](start = 19, length = 5)

we havent released 2.1.3 yet! so this change will go as part of 2.1.3, we should revert this change

Reply:
Done.

---
In reply to: [290513039](https://github.com/microsoft/sarif-sdk/pull/1519#discussion_r290513039) [](ancestors = 290513039)